tools/timemachine/internal/cmd/download.go

```diff
@@ -0,0 +1,219 @@
+// Copyright (c) 2022 IoTeX Foundation
+// This source code is provided 'as is' and no warranties are given as to title or non-infringement, merchantability
+// or fitness for purpose and, to the extent permitted by law, all liability for your use of the code is disclaimed.
+// This source code is governed by Apache License 2.0 that can be found in the LICENSE file.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path"
+	"path/filepath"
+	"sort"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"google.golang.org/api/iterator"
+
+	"github.com/iotexproject/iotex-core/pkg/log"
+)
+
+// const
+const (
+	_gcpTimeout = time.Second * 60
+	_bucket     = "blockchain-golden"
+	_objPrefix  = "fullsync/mainnet/"
```

Comment:
support both mainnet and testnet. you can leave it to next pr

Reply:
fixed